tests/unit/pages/quiz-attempt/QuizAttempts.spec.jsx

```diff
@@ -56,10 +86,16 @@ describe('QuizPage for student', () => {
         new RegExp(URLs.finishedQuizzes.getById.replace(':id', mockQuizId))
       )
       .reply(200, mockQuiz)
+
+    mockAxiosClient
+      .onGet(
+        '/finished-quizzes//'
```

Comment:
```suggestion
        '/finished-quizzes/'
```
I guess there is an extra slash.

Reply:
No there isn't im not sure why but when I do a console log in base service it does requests to finished-quizzes// and im not sure how to fix it :/

Reply:
Inside your component, probably, there are `initialValues` that have an `id` as `''`. So instead of a real `id` from mock, it consumes just an empty string as id.
Check that, please, because it's not how it should work.

Maybe you should mock something, I don't know.